src/clims/configuration/hooks.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function
+
+HOOK_TYPE = '_hook_type'
+
+HOOK_TAG = '_hook_tag'
+
+
+def button(button_text):
```

Comment:
I think the idea of having a decorator for this is the way to go, as we'll both have a way to describe the UI as well as handling the event in the same place, which follows the api-simplicity design goal we have :+1:

Reply:
:+1: